docs/examples/general/expressions/expr_examples.ipynb

```diff
@@ -4,26 +4,32 @@
    "cell_type": "markdown",
```

Comment:
<p><code>The constant operand for arithmetic operator can be one of the following options:  </code></p><ul><li><code>Values of Python's&nbsp;int&nbsp;and&nbsp;float&nbsp;types that are used directly.</code></li><li><code>Values that are wrapped in&nbsp;nvidia.dali.types.Constant.</code></li></ul><p> </p><p>Is malformed in html, no bullet points there.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/2480/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Done</p>